include/swift/AST/Types.h

```diff
@@ -4035,12 +4043,13 @@ class SILFunctionType final : public TypeBase, public llvm::FoldingSetNode,
   unsigned NumAnyResults : 16;         // Not including the ErrorResult.
   unsigned NumAnyIndirectFormalResults : 16; // Subset of NumAnyResults.
 
+  // [SILFunctionType-layout]
   // The layout of a SILFunctionType in memory is:
   //   SILFunctionType
   //   SILParameterInfo[NumParameters]
   //   SILResultInfo[isCoroutine() ? 0 : NumAnyResults]
-  //   SILYieldInfo[isCoroutine() ? NumAnyResults : 0]
   //   SILResultInfo?    // if hasErrorResult()
+  //   SILYieldInfo[isCoroutine() ? NumAnyResults : 0]
```

Comment:
Should this be `NumYields` instead of `NumAnyResults`?

Reply:
If `isCoroutine()` is true, then `NumAnyResults` is `NumYields` and if it is false, then it is equal to `NumResults`. Since there are no explicit fields `NumYields` or NumResults` (unlike `NumParameters`), it seems a bit misleading to refer to them.